docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '2.1'
+services:
+  docs:
+    build:
+      context: ./src/angularjs
+      dockerfile: Dockerfile
+    working_dir: /opt/angularjs
+    volumes:
+     - ./src/angularjs:/opt/angularjs
```

Comment:
Despite the ugliness, I think that we should setup the volumes here like we do in the main Raster Foundry project (one for each file at the top level, and then another two for source and `node_modules`).